RecoTauTag/RecoTau/plugins/DeepTauId.cc

```diff
@@ -584,7 +584,9 @@ class DeepTauId : public deep_tau::DeepTauBase {
 
     static double getInnerSignalConeRadius(double pt)
     {
-        return std::max(.05, std::min(.1, 3./std::max(1., pt)));
```

Comment:
A few comments on signal cone size formula:
1. Original formula is as follows `std::max(std::min(0.1, 3.0/pt), 0.05)`;
2. You replaced `pt` by `std::max(1., pt)` in the formula it improve numerical stability if I understand correctly, right? In principle one can use pt_min of 30 instead of 1;
3. Current version of the formula is even better, but I would add a comment that it is equivalent of the original formula (`std::max(std::min(0.1, 3.0/pt), 0.05)`), use the tighter `min_pt` of 30, and do not reset value of pt, but put `std::max(pt, min_pt)` inside `std::clamp`.

Reply:
The "original" formula on which it is based is taken from here:
https://github.com/cms-sw/cmssw/blob/CMSSW_9_4_X/RecoTauTag/RecoTau/src/AntiElectronIDMVA6.cc#L575
and yes, I suppose that `max(1., pt)` was introduced there to improve a numerical stability. Putting 
min pt 30 GeV would be too much because the deetTauId discriminator is trained with taus > 20 GeV. So, I propose or to keep 1 GeV (to be compatible with the "original" formula) or to put 10 GeV (to keep it below any possible threshold for the foreseeable future).

Reply:
Oh, sorry, you right: there is a `min(0.1, 3.0 / pt)`. So yes, then the code can be simplified this way:
`return std::max(cone_opening_coef / std::min(pt, min_pt, min_radius);` with min_pt = 30.